wp-content/themes/illdy-child/layout/js/translations_page.js

```diff
@@ -82,18 +82,32 @@ var TranslationsPage = (function(window, $) {
       // Give sidebar the list of languges
       this.sidebar.setList(this.translationsData);
 
-      // Pre-select a language if there's something in the query string
-      // TODO: Look for the 'lang' parameter on the query string specifically
+      // Default to English resources. Will trigger sidebar.render();
+      this.sidebar.setSelected('en');
+
+      // Process search string
       var searchString = window.location.search;
       if (searchString) {
-        var langSearch = searchString.split('=')[1];
-        // Will trigger sidebar.render();
-        this.sidebar.setSelected(langSearch);
-        return;
-      }
 
-      // Default to English resources. Will trigger sidebar.render();
-      this.sidebar.setSelected('en');
+        // Pre-select a language if given
+        var langRegex = new RegExp("\\Wlang=([^&]+)");
+        var result = langRegex.exec(searchString);
+        if (result && result.length >= 2) {
+            var langSearch = result[1];
+            // Will trigger sidebar.render();
+            this.sidebar.setSelected(langSearch);
+        }
+
+        // Pre-select a resource if given

```

Comment:
If only IE had support for URLSearchParams  https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams